<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesDialog</class>
 <widget class="QDialog" name="FilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listFiles">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>45</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFiles">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFolders">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>310</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Folders</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listFolders">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>340</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddFile">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>45</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoveFile">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>85</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearFile">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>125</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddFolder">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>340</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoveFolder">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>380</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearFolder">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>420</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackups">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>500</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Backups:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetBackups">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>525</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Resets the backup counter back to 0.
This affects the name of the folders created when backing up.
Only reset if you cleaned all previous backups or they'll be overwritten.</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputBackup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>525</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The amount of times this preset was backed up</string>
   </property>
   <property name="text">
    <string notr="true">0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>560</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
